src/Pact/Analyze/Analyze.hs

```diff
@@ -339,44 +316,6 @@ mkSymbolicCells tables = TableMap $ Map.fromList cellsList
 mkSVal :: SBV a -> SBVI.SVal
 mkSVal (SBVI.SBV v) = v
 
-describeAnalyzeFailureNoLoc :: AnalyzeFailureNoLoc -> Text
-describeAnalyzeFailureNoLoc = \case
```

Comment:
Glad to be moving stuff into new files. We have too much stuff in too few files atm